corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -80,7 +79,7 @@ def interactive_import(self):
     @memoized
     def jenkin_projects(self):
         print("Pinging Jenkins build server. Pelase wait...")
-        return list(six.iterkeys(self.build_server))
+        return list(self.build_server)
```

Comment:
shucks! the one place I would point this out the class is not actually a `dict`. `self.build_server` is an instance of `Jenkins`, which has no `__iter__` method, so this will not work. Very sorry, but you'll have to change it back
```suggestion
        return list(self.build_server.keys())
```

Reply:
I went ahead and committed this suggestion

Reply:
Thank you :)

I'm enjoying this pattern where I do some fairly brain-dead python 3 work while waiting for tests to run or during some other garbage time, and then you and @kaapstorm smooth out the rough edges. I think we're making good progress.